src/Article.jsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react"
+import { useParams } from "react-router-dom"
+
+function Article(){
+
+    const [article, setArticle] = useEffect({})
```

Comment:
i know this isn't the correct ticket but did you mean useEffect

Reply:
Yes typo